web-analogs.md

```diff
@@ -162,8 +162,11 @@ var container = new Container( // grey box
 ### Setting Container Width
 
 To specify the width of a [Container](https://docs.flutter.io/flutter/widgets/Container-class.html)
-widget you set its ```width``` property. This is equivalent to specifying the CSS 
-max-width property.
+widget you set its ```width``` property. This is a static width, unlike the 
```

Comment:
you only need single back-ticks for inline code. triple backticks are useful for multi-line code.